code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -735,6 +775,7 @@
 /datum/tarot/reversed/justice
 	name = "VIII - Justice?"
 	desc = "May your sins come back to torment you."
+	extended_desc = "creates a random crate from cargo. This can include stuff they would otherwise not have access to at the time."
```

Comment:
```suggestion
	extended_desc = "creates a random cargo crate. This can include crates the user would otherwise not have access to at the time."
```

Reply:
worded it slightly differently to how you have, as it doesn't *give* the user access to the crate, but i can summon things like the BSH crate on a round where the goal isn't the BSH.